.github/workflows/CD.yml

```diff
@@ -58,15 +58,33 @@ jobs:
 
     if: ${{ !github.event.repository.fork }}
 
+    env:
+      BOT_HOME_DIR: /home/csc
+
     steps:
+    - uses: actions/checkout@v4
+      with:
+        show-progress: false
+
+    - name: Copy infra directory
+      uses: appleboy/scp-action@v0.1.7
+      with:
+        host: ${{ secrets.SSH_HOST }}
+        username: ${{ secrets.SSH_USERNAME }}
+        key: ${{ secrets.SSH_KEY }}
+        port: ${{ secrets.SSH_PORT }}
+        source: .infra
+        target: ${{ env.BOT_HOME_DIR }}
+
+
     - name: Deploy CSZ Bot
       uses: appleboy/ssh-action@029f5b4aeeeb58fdfe1410a5d17f967dacf36262
       env:
-        DOCKER_BUILDKIT: 1
+        WORKING_DIRECTORY: ${{ env.BOT_HOME_DIR }}
       with:
         host: ${{ secrets.SSH_HOST }}
         username: ${{ secrets.SSH_USERNAME }}
         key: ${{ secrets.SSH_KEY }}
         port: ${{ secrets.SSH_PORT }}
-        envs: DOCKER_BUILDKIT
-        script: /home/csc/deploy.sh
+        envs: WORKING_DIRECTORY
+        script: ${{ env.BOT_HOME_DIR }}/deploy.sh
```

Comment:
vielleicht kann man das deploy-skript auch ersetzen, indem man im CI mit einem docker-context arbeitet. Der kann auch SSH
<https://blog.mikesir87.io/2019/08/using-ssh-connections-in-docker-contexts/>

kA ob das hier passt